src/test/cypress/support/pageobjects/ArtemisPageobjects.ts

```diff
@@ -47,6 +52,13 @@ export class ArtemisPageobjects {
         exampleSubmissions: new TextExerciseExampleSubmissionsPage(),
         exampleSubmissionCreation: new TextExerciseExampleSubmissionCreationPage(),
         editor: new TextEditorPage(),
+        feedback: new TextExerciseFeedbackPage(),
+    };
+    assessment = {
+        dashboard: new AssessmentDashboard(),
+        course: new CourseAssessmentDashboardPage(),
+        exercise: new ExerciseAssessmentDashboardPage(),
+        text: new TextExerciseAssessmentPage(),
```

Comment:
Have you gone through all existing spec files and changed the references to these pageobjects where necessary?

Reply:
No, I think think the `AssessmentDashboard` is even a duplicate, because it comes from another branch. I will merge it with the other pageobjects

Reply:
I adjusted the ExamAssessment spec to use the new pageobjects and refactored the old pageobject